signer/src/block_observer.rs

```diff
@@ -155,6 +156,11 @@ where
                         tracing::warn!(%error, "could not load latest deposit requests from Emily");
                     }
 
+                    if let Err(error) = self.update_sbtc_limits().await {
```

Comment:
I think maybe this should be moved above the `load_latest_deposit_requests` block above.

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/811/commits/086c1a9a380d8416c63ea1edca2d3307453a0bf1